.github/workflows/commitlint.yaml

```diff
@@ -0,0 +1,26 @@
+---
+name: Commitlint
+on:
+  push:
+    branches-ignore:
+      - 'main'
```

Comment:
String is the default data type in YML so i believe the quotes are not needed.

Reply:
Ok, quotes removed.